folder/Kptfile

```diff
@@ -0,0 +1,48 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: folder
+openAPI:
+  definitions:
+    io.k8s.cli.setters.name:
+      x-k8s-cli:
+        setter:
+          name: name
+          value: ${FOLDER_NAME}
+          required: true
+      description: folder name
+    io.k8s.cli.substitutions.folder-name:
```

Comment:
You shouldn't need a substitution here—the setter can be referenced directly.

Reply:
using a prefix in folder name